### PR TITLE
Edit content for new or prospective VSF contributors 

### LINKED ISSUE #258: Edit content for new or prospective VSF contributors
Overview of the VSF activities for new members (and also to simplify onboarding.

* Add to the webpage a "get involved" page that describes what we do and how people can join.
* Review the "call for organizers page"
* Currently, the contact-us only lists communication means.

### PR DESCRIPTION
**Pull request by everthemore**
closes #258

### CODE REVIEW

**Comment by everthemore** [comment]
closes #258

**Comment by akhmerov** [comment]
I think a more concrete document describing what we actually do would be useful to the potential reader.

Something along the lines "We organize colloquia, develop and improve this website, and host events. We meet weekly to discuss, join anytime."

**Comment by akhmerov** [comment]
i.e. giving an idea into what tasks we invest our time.

**Comment by everthemore** [comment]
Close this in favor of #304?

**Comment by akhmerov** [comment]
The call for organizers still seems useful, right?

**Comment by everthemore** [comment]
But do we want to keep this explicit call, rather than having a general 'join us'? The suggestions you made have now been added to contact.md, perhaps we should just link there instead?

File: home.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ iframe.twitter-follow-button {
 </style>
 <a href="https://twitter.com/virtualsciforum?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-dnt="true" data-show-count="true">Follow @virtualsciforum</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 
+### Join us
+
+We are always happy to welcome new members on the organizational side of VSF. 
+Please take a look at our [call for organizers page](announcement.md) for more information.
```
Comment (greplova) [inline home.md:36]: Please take a look [here](https://virtualscienceforum.org/#/contact) to see how to keep up with our activities or join our organising team.

Comment (greplova) [reply home.md:36]: I think this is great - I just modified the sentence to link to the contact us page.

**Comment by everthemore** [comment]
Good to merge @akhmerov?

**Review by akhmerov — APPROVED** [review]

**Comment by akhmerov** [comment]
Yep, looks good.

**Pull request merged**